### ISSUE TITLE
[Bug] can not drop multi column in one next

### ISSUE THREAD

**Issue by xialvjun** [Ff00ff/mammoth#196]
```ts
export const foo = m.defineTable({
  id: m.uuid().primaryKey().default(`gen_random_uuid()`),
  name: m.text().notNull(),
  value: m.integer(),
});
```

then

```sh
mammoth next --dbFile=src/db.js
```

got

```sql
CREATE TABLE foo (
  id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
  name text NOT NULL,
  value pg_catalog.int4
);
```

then change ts

```ts
export const foo = m.defineTable({
  id: m.uuid().primaryKey().default(`gen_random_uuid()`),
});
```

then next again, then got

```sql
ALTER TABLE foo DROP COLUMN name;
```

besides, another bug, `mammoth-cli` default `dbFile` should be `db.js`, otherwise `SyntaxError: Cannot use import statement outside a module`

**Comment by martijndeh** [comment]
Check, I'll try and reproduce this in the coming days.

**Comment by martijndeh** [comment]
> besides, another bug, mammoth-cli default dbFile should be db.js, otherwise SyntaxError: Cannot use import statement outside a module

This is fixed in the latest cli release. You should just be able to use any TypeScript file exporting your db. The issue was that tsconfig.json was not included in the published files and so ts-node didn't properly compile the references TypeScript file.

I'll have a look at the column drop issue next.

**Comment by martijndeh** [comment]
Dropping multiple columns in one next is now also supported. The fix is released in `@ff00ff/mammoth-cli@1.0.0-beta.8`.

Thanks for the report. It was very easy to reproduce this way!
